_episodes/030-krustlet.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "WebAssembly on the Server with Krustlet" 
+date: 2020-09-22T00:50:00+00:00
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e030-krustlet.mp3
+duration: "01:02:55"
+length: "59139822"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+
+---
+
+Taylor Thomas explains how Krustlet runs WebAssembly modules in Kubernetes and why it's a promising option for the future of server side applications.
+
+### Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to be interviewed, propose a topic for an episode, or help create the podcast itself!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps 
+
+- [@00:55] - Kubernetes
+- [@07:37] - WebAssembly
+- [@12:06] - WebAssembly Runtimes and WASI Specification
+- [@15:42] - WebAssembly vs Containers vs Native Binaries
+- [@25:11] - Krustlet and the case for writing it in Rust
+- [@30:52] - Missing APIs in WASI 
+- [@33:38] - Wascc vs Wasmtime runtimes
+- [@38:15] - Rust ecosystem for Kubernetes and WebAssembly
+- [@40:23] - Comparing other languages to Rust
+- [@45:09] - Rust learning curve, experiences as a beginner
+- [@53:16] - Next steps for Krustlet and WebAssembly
+
+### Referenced Resources
+
+- [Krustlet](https://github.com/deislabs/krustlet)
+- [Kubernetes](https://kubernetes.io/)
+- [Open Container Initiative](https://opencontainers.org/)
+- [WebAssembly](https://webassembly.org/)
+- [WASI](https://wasi.dev/)
+- [Wasmtime](https://wasmtime.dev/)
+- [waSCC](https://wascc.dev/)
+- [WebAssembly meets Kubernetes with Krustlet](https://cloudblogs.microsoft.com/opensource/2020/04/07/announcing-krustlet-kubernetes-rust-kubelet-webassembly-wasm/)
+- [Introducing Krustlet, the WebAssembly Kubelet](https://deislabs.io/posts/introducing-krustlet/)
+- [Kubernetes: A Rusty Friendship](https://deislabs.io/posts/kubernetes-a-rusty-friendship/)
+- [The Safety Boat: Kubernetes and Rust](https://msrc-blog.microsoft.com/2020/04/29/the-safety-boat-kubernetes-and-rust/)
+- [A Heaping Helping of Stacks](https://deislabs.io/posts/a-heaping-helping-of-stacks/)
+
+### Credits
+
+Intro Theme: [Aerocity](https://twitter.com/AerocityMusic)
+
+Audio Editing: [Jeremy Jung](https://www.softwaresessions.com)
+
+Hosting Infrastructure: [Jon Gjengset](https://twitter.com/jonhoo/)
+
+Show Notes: Jeremy Jung 
+
+Hosts: Jeremy Jung
+
+Guests: [Taylor Thomas](https://twitter.com/_oftaylor)
```

Comment:
I would probably link to the guest's twitter as a link further up in the description as well :)

Reply:
i added it to the guest name at the top, is that what you were thinking?